x-pack/plugins/ml/public/application/services/ml_api_service/saved_objects.ts

```diff
@@ -28,7 +28,7 @@ import type {
 export const savedObjectsApiProvider = (httpService: HttpService) => ({
   jobsSpaces() {
     return httpService.http<JobsSpacesResponse>({
-      path: `${basePath()}/saved_objects/jobs_spaces`,
+      path: `${ML_INTERNAL_BASE_PATH}/saved_objects/jobs_spaces`,
       method: 'GET',
```

Comment:
Is version needed here?

Reply:
10 points for discovering the deliberate mistake!
Route updated in https://github.com/elastic/kibana/pull/156949/commits/df24ed46c5ba250cff41580fb85d6f569ad1fc5e